src/.clang-format

```
@@ -0,0 +1,30 @@
+BasedOnStyle: LLVM
+IndentWidth: 4
+TabWidth: 4
+UseTab: Never
+ColumnLimit: 120
+PenaltyBreakComment: 300
+PenaltyBreakFirstLessLess: 120
+PenaltyBreakString: 100
+PenaltyExcessCharacter: 100
+MaxEmptyLinesToKeep: 2
+BreakBeforeBraces: Attach
+AllowShortIfStatementsOnASingleLine: true
```

It doesn't seem like `true` is one of the possible values of this config. https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortifstatementsonasingleline

If we want to allow it for if without else, we could use

```suggestion
AllowShortIfStatementsOnASingleLine: WithoutElse
```

---

I think we should also enable

    AllowShortCaseLabelsOnASingleLine: true